src/_articles/language/mixins.md

```diff
@@ -2,18 +2,27 @@
 title: A Brief History of Mixins in Dart
 description: Mixins let you implement functionality once and use it in multiple classes.
 original-date: 2012-12-18
-date: 2018-09-21
+date: 2018-06-01
 category: language
+obsolete: true
 ---
 
 _Written by Gilad Bracha <br>
 December 2012 (updated June 2018)_
 
-Looking for an introduction to mixins?
-See the Language Tour section [Adding features to a class: mixins][].
+This article provides a brief theoretical presentation of
+the evolution of mixins in Dart.
 
```

Comment:
Do you want to keep the old link to the lang tour?  E.g., something like: "For a guide on how to use mixins today, please see ... instead" or similar?

Reply:
I moved that link to a note, hoping it'd be more noticeable. Did you look at the staged page?

https://kw-www-dartlang-1.firebaseapp.com/articles/language/mixins

Reply:
got it - thanks!